ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly-4.13-upgrade-from-stable-4.12.yaml

```diff
@@ -466,8 +466,8 @@ tests:
     - ref: cucushift-installer-check
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-vsphere-ipi-disk-encryption-tang
-- as: vsphere-ipi-external-lb-p2-f14
-  cron: 36 23 11,27 * *
+- as: vsphere-ipi-external-lb-post-p2-f14
```

Comment:
Per the update, we also need to update all the other job name for post case?

Reply:
i think so.  the previous workflow was cucushift-installer-rehearse-vsphere-ipi-external-lb, now it's cucushift-installer-rehearse-vsphere-ipi-external-lb-post.  I just found there are still two files need to be update with new workflow. let me update them accordingly.

./ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12-periodics.yaml
./ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml

Reply:
Nice!

Reply:
if no issue, please help approve and merge the pr @jianlinliu